### PR TITLE
MenuBox Component

### PR DESCRIPTION
**Pull request by krumaska**
This Component Using in Menu Page
imgPath : `assets/{imgPath}`
textName : MenuBox Name

### CODE REVIEW

**Comment by hsmint** [comment]
2nd and 3rd box not showing up center?

File: src/components/MenuBox.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<template>
+	<div class="container" >
+		<img :src="require(`@/assets/${imgPath}`)">
```
Comment (CXZ7720) [inline src/components/MenuBox.vue:3]: is `require` works here?
what about remoe it and just using image path like this code?
https://github.com/trade-dog/web_front/blob/5c02624fa699a9a4cefed686d9a4072cfcfc67b2/src/components/TrendCard.vue#L28

Comment (krumaska) [reply src/components/MenuBox.vue:3]: Thank you for suggestion. 👍 
`require` works here.
I tested `<MenuBox imgPath="gubean.jpg"...></MenuBox>`
with this code `` in views.  
Unfortunately, this code did not work.

**Comment by krumaska** [comment]
@hsmint  this MenuBox Components not showing up center, actually...  
If it overlaps with the first menu box, it will showing up center.

File: src/components/MenuBox.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		width: 100%;
+		opacity: 0.65;
+		position: absolute;
+	}
+	.container .text-block {
+		position: absolute;
+		display: flex;
+		height: 120%;
+		left: 10%;
+		text-align: center;
```
Comment (hsmint) [inline src/components/MenuBox.vue:45]: Maybe you should change the style here to make sure it stays center whatever the size of the box is... flexible box might help

File: src/components/MenuBox.vue (hunk 1 above)
Comment (CXZ7720) [reply src/components/MenuBox.vue:3]: Okay. I'll check about about it after merged into `Menu` branch at this repo.
This issue will be treated another PR before merged into master branch.

File: src/components/MenuBox.vue (hunk 2 above)
Comment (krumaska) [reply src/components/MenuBox.vue:44]: I removed text-block height due to customize.

**Pull request merged**